### ISSUE TITLE
Change License to MIT

### ISSUE THREAD

**Issue by Abdera7mane** [openmindsclub/Recipes#1]
I believe food recipes should be be under a permissive license, so they don't serve the meal with a receipt next to it (copy left).

## Why ?

We can’t get around it, so we’ll say it upfront. Food is essential to life. What’s more, ensuring open access to the resources, knowledge, and land we need to feed ourselves is political. In opposition to corporate control and intellectual property, we need systems and processes which emphasize sharing and collaboration for food systems work.

A movement of practitioners is emerging who are applying Open Source philosophy to food systems work. Our goal in this article is to recognize the power of this fledgling movement and to highlight some of the people and organizations who are using Open Source for food.

For food systems work, Open Source means open ideas, knowledge, plans, documents, tools, code, data, and so on, all open for use and improvement by others. Instead of privatizing and patenting intellectual property, we’re sharing designs and building off each other’s innovations.

When source code is “open”, anyone can start where someone else left off. Any documented piece of knowledge can be reused and redirected to fit unique situations and local needs. It can be shared across the globe and implemented quickly and easily by local people and organizations working within their communities

**Comment by cr105ph1nx** [comment]
TLDR Git rekt LMAO !

**Comment by Abdera7mane** [comment]
How rude

**Comment by Abdera7mane** [comment]
I will revert your PR
